### PR TITLE
Basics concept

### PR DESCRIPTION
**Pull request by rlmark**
Taking a stab at adding a basics concept for Unison with associated lasagna exercise.

I've added it as "wip" here

### CODE REVIEW

**Review by kytrinyx — APPROVED** [review]
This looks great! I think it's good to be merged, since any improvements and tweaks can easily be made in a second PR.

One thing I'd recommend is having someone on the team try to solve the exercise given only the instructions (not the test suite). This is a quick way of figuring out if anything is missing in the instructions.

Once it's merged and `wip` on the site, I'll try solving it as a student. I've never done any Haskell or Unison so I'm a very good naive subject :)

File: exercises/concept/lasagna/.docs/instructions.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Instructions
+
+In this exercise you're going to write some code to help you cook a brilliant lasagna from your favorite cooking book.
+
+You have three tasks, all related to the time spent cooking the lasagna.
+
+## 1. Define the expected oven time in minutes
+
+Define the `expectedMinutesInOven` constant to check how many minutes the lasagna should be in the oven. According to the cooking book, the expected oven time in minutes is 40:
```
Comment (ErikSchierboom) [inline exercises/concept/lasagna/.docs/instructions.md:9]: Should this be:

```suggestion
Define the `expectedMinutesInOven` term to check how many minutes the lasagna should be in the oven. According to the cooking book, the expected oven time in minutes is 40:
```

File: exercises/concept/lasagna/.docs/introduction.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+Unison functions look like `myFunctionName argument1 argument2 = myImplementation`; they start with the name of the function followed by space-separated arguments.
+
+```
+getMax a b = Nat.max a b
+```
+
+Functions can optionally include a type signature. The last type on the right is always the final return type of the function, and each argument is separated by the function arrow `->`.
+
+```
+{{getMax accepts two arguments of type Nat and its return type is a Nat}}
```
Comment (ErikSchierboom) [inline exercises/concept/lasagna/.docs/introduction.md:25]: Up until this point, the documentation literal syntax is not explained. You could consider omitting it here and only have it later on, when the syntax is explained.

**Review by ErikSchierboom — APPROVED** [review]
Great work! Just some minor nits.

**Pull request merged**